product_packaging_barcode/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Packaging Barcode',
+    'description': """
+        Reimplement barcode field in Product Packagings (was deleted in v9)""",
```

Comment:
Remove this description, as it's taken from README.rst. Maybe you wanted to put `summary`?